### DISCUSSION TITLE
How to sync authenticated users of Clerk into Supabase DB automatically without any extra frontend or backend codes update.

### DISCUSSION THREAD

**Discussion by IntelMin** [supabase/supabase#19098] [Questions]
Hello devs,
I am wondering if it is possible to sync authenticated users data from Clerk into Supabase DB automatically without any extra frontend or backend codes.

So I have a website that the auth is implemented using Clerk service, and I still like that way.
I am just trying to sync all the auth data into Supabase because I have another logic there.
- add newly signed up users to Supabase DB
- update modified users to Supabase DB
- remove deleted accounts from Supabase DB
...

I hope to achieve this goal by not making extra codes updates outside of these two services, but just using some original features of Clerk and Supabase only, such as webhooks. But I noticed that webhooks are designed to call a http url when it triggers, and I have no idea like to serve that kind of url on Supabase, and it is impossible to add my own code blocks into Supabase backend, for example.
Because I don't want to modify frontend or backend codes at all.
Just want to configure Two Services only to make things work!

Is there anyone to give me a clear idea to achieve this, or is it even impossible to do?
Thanks.

**Comment by HamedMP** [comment]
Have you been able to find a solution for this? I also need to do it and see Webhooks as the only option.

**Comment by krishna-praveen** [comment]
I am trying to do the same thing and ultimately i found out that this is the issue with third party auth providers. I am unsure how people actually use third party auth's when we have issues like this.

**Comment by IntelMin** [comment]
I have not found the proper solution for this yet. Eventually, I did it by making additional codes for syncing, on my separate nodejs backend.
However always hope to figure out the solution if it might be still possible.

**Comment by ryanknight09** [comment]
web hooks are the way to go. Just add it to your code base to listen for user events like create user or update user, then take the payload and do an insert or update to your supabase db.

**Comment by benoitabijuru** [comment]
I found clerk guide for clerk and supabase user sync helpful, check this link , https://clerk.com/blog/sync-clerk-user-data-to-supabase
